vms/vms/settings.py

```diff
@@ -61,8 +61,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'vms',
-        'USER': 'myuser',
-        'PASSWORD': 'mypassword',
+        'USER': 'vmsadmin',
+        'PASSWORD': '0xdeadbeef',
         'HOST': 'localhost',
```

Comment:
If its intentional to leave it that way, then its fine.

Btw, I attended the Codeathon @ Grace Hopper 2015 last week. Still not
familiar with the standards followed by this project.


On Mon, Oct 19, 2015 at 7:09 AM, Tapasweni Pathak <notifications@github.com>
wrote:

> In vms/vms/settings.py
> <https://github.com/systers/vms/pull/155#discussion_r42363204>:
>
> > @@ -61,8 +61,8 @@
> >      'default': {
> >          'ENGINE': 'django.db.backends.postgresql_psycopg2',
> >          'NAME': 'vms',
> > -        'USER': 'myuser',
> > -        'PASSWORD': 'mypassword',
> > +        'USER': 'vmsadmin',
> > +        'PASSWORD': '0xdeadbeef',
> >          'HOST': 'localhost',
>
> Isn't this what the person who is installing choose? So instead of myuser
> and mypassword you need to type your own.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/systers/vms/pull/155/files#r42363204>.
>